maquetacion/src/scss/_detail.scss

```diff
@@ -0,0 +1,50 @@
+.detail {
+  background-color: #FFF;
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+  display: flex;
+  height: 416px;
+  max-width: 900px;
+  padding: 24px;
+  width: 100%;
+
+  &__content {
+    width: 100%;
+  }
+
+  &__cover {
+    margin-right: 52px;
+  }
+
+  &__cover-img {
+    height: 368px;
+    width: 261px;
+  }
+
+  &__genre {
+    height: 30px;
+    width: 98px;
+    font-family: Roboto;
```

Comment:
I think everything has Roboto, if not remember to set a backup like `sans-serif`

Reply:
Fixed in last branch / commit